backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -630,6 +630,7 @@ private void deleteRepositories(Session session) {
                               ModelDBConstants.DEFAULT_VERSIONING_BLOB_LOCATION));
                   deleteLabels(session, compositeId, IDTypeEnum.IDType.VERSIONING_REPO_COMMIT_BLOB);
                   deleteAttribute(session, compositeId);
+                  deleteTagEntities(session, repository.getId(), commitEntity.getCommit_hash());
```

Comment:
Please extract this in different PR , this is a fix worthy to be in master

Reply:
Removed from here and created new PR: https://github.com/VertaAI/modeldb/pull/920